### ISSUE TITLE
Feature proposal

### ISSUE THREAD

**Issue by empusaee** [julien-nc/cospend-nc#186]
Let's say there is a household with many mixed shopping items, which could be managed inside a project. 
But then one project member maybe wants to have some transactions that the others should not see
Currently there is no privacy option to hide personal transactions from other members, only the possibility to create a new project for personal use...which is also tricky because each project requires a minimum of 2 members
Either way, but by doing that, there is no way of merging statistics.
So one awesome feature would be either to be able to merge statistics on multiple projects or add a privacy field to records. The latter is preferable.

**Comment by mhastu** [comment]
I agree, an overall statistic would be nice. But I think the system of different projects is enough for privacy. E.g. I have a "personal" project with only me as a member (there was no minimum member count for me). I manually create a final bill summarizing a shared project like a vacation in my personal project. This way, I have statistics of just my personal finances.

It would be nice if this process could be automated, e.g. by importing bills of another project and providing mappings for payment methods and categories.
Even better would be a "global statistics" feature, where the user can configure the projects to use, and the category and payment method mappings. This might be implemented as a "smart project" feature, where the bills of all given projects are automatically imported.
